package-lock.json

```diff
@@ -7457,6 +7471,20 @@
       "resolved": "https://registry.npmjs.org/react-gtm-module/-/react-gtm-module-2.0.11.tgz",
       "integrity": "sha512-8gyj4TTxeP7eEyc2QKawEuQoAZdjKvMY4pgWfycGmqGByhs17fR+zEBs0JUDq4US/l+vbTl+6zvUIx27iDo/Vw=="
     },
+    "node_modules/react-helmet": {
```

Comment:
As I see your PR try to add not needed dependencies. Have you tried installing any additional dependencies? Because `package-lock.json` should remain intact.